addons/purchase/models/purchase_order.py

```diff
@@ -146,6 +146,7 @@ def _compute_invoice(self):
     company_id = fields.Many2one('res.company', 'Company', required=True, index=True, default=lambda self: self.env.company.id)
     company_currency_id = fields.Many2one(related="company_id.currency_id", string="Company Currency")
     country_code = fields.Char(related='company_id.account_fiscal_country_id.code', string="Country code")
+    company_price_include = fields.Selection(related='company_id.account_price_include', readonly=True)
```

Comment:
```suggestion
    company_price_include = fields.Selection(related='company_id.account_price_include')
```
(a related is readonly by default)